Apps/phciscothreatresponse/ciscothreatresponse.json

```diff
@@ -0,0 +1,543 @@
+{
+    "appid": "21e986c0-6855-4ab0-89a9-080f92f323dc",
+    "name": "Cisco Threat Response",
+    "description": "Cisco Threat Response app for Phantom provides actions allowing users to query Cisco Threat Response for verdicts and context on observables within a Phantom instance",
+    "type": "information",
+    "product_vendor": "Cisco Security",
+    "logo": "logo_ciscothreatresponse.svg",
+    "logo_dark": "logo_ciscothreatresponse_dark.svg",
+    "product_name": "Cisco Threat Response",
+    "product_version_regex": ".*",
+    "publisher": "Splunk",
+    "license": "Copyright (c) 2020 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2019-11-25T14:17:29.991850Z",
+    "package_name": "phantom_ciscothreatresponse",
+    "main_module": "ciscothreatresponse_connector.pyc",
```

Comment:
Please add `latest_tested_versions`

Reply:
@alexa-phantom It is a community app. So we cannot add the latest_tested_versions